src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/WindowsRuntime/Generated/WinRT.cs

```diff
@@ -0,0 +1,289 @@
+// This file was generated by cswinrt.exe version 0.0.0-private.0
```

Comment:
- Add copyright
- if it is generated then can it be generated during build and consumed from OBJ dir? Checking in a generated file should be avoided if possible. 
- describe generation steps in comments in more detail if possible

Reply:
Same comments for all generated files

Reply:
These files cannot be generated during build since they've needed at least partial modifications to internal-ize public surface area and remove unused types. These should be considered a one-off generation. We'll remove this once the Windows SDK is ready to ship as part of the TFM.

I'll add the license header.